### PR TITLE
support to balance subtasks to new captures

### LINKED ISSUE #638: Changefeed subtasks can not be balanced to new captures
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### What did you do? If possible, provide a recipe for reproducing the error.

Suppose we have a changefeed that needs to sync 2 tables (subtasks) running on a cdc server (capture).
After we adding a new capture, subtasks can not be balanced to the new capture. 

### What did you expect to see?

Subtasks are balanced to two captures evenly.

### What did you see instead?

Does not balance.

### Versions of the cluster

- Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        v4.0.0

- TiCDC version (execute `cdc version`):

        v4.0.0

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
support to balance subtasks to new captures ,fix #638

### What is changed and how it works?
- Adjust the default value of `Scheduler.PollingTime` to `-1`
- If `Scheduler.PollingTime` is less than 0, the rebanlance process will not be call by time trigger.

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- support to balance subtasks to new captures

### CODE REVIEW

File: cdc/changefeed.go [hunk 1]
```diff
@@ -429,9 +429,10 @@ func (c *changeFeed) rebanlanceTables(ctx context.Context, captures map[model.Ca
 			return nil
 		}
 	}
+	timeToRebanlance := time.Since(c.lastRebanlanceTime) > time.Duration(c.info.Config.Scheduler.PollingTime)*time.Minute
+	timeToRebanlance = timeToRebanlance && c.info.Config.Scheduler.PollingTime > 0
 
-	if !c.rebanlanceNextTick &&
-		time.Since(c.lastRebanlanceTime) < time.Duration(c.info.Config.Scheduler.PollingTime)*time.Minute {
+	if !c.rebanlanceNextTick && !timeToRebanlance {
```
Comment (amyangfei) [inline cdc/changefeed.go:435]: Supposing we add a capture, the rebalance will be not automatically triggered

Comment (leoppro) [reply cdc/changefeed.go:435]: I will add a test for it

**Pull request merged**